apps/src/templates/instructions/ScrollButtons.jsx

```diff
@@ -107,95 +118,103 @@ class ScrollButtons extends React.Component {
       ? this.props.height > 100
       : this.props.height > 60;
 
-    let upStyle = {
-      opacity: this.props.visible ? 1 : 0,
-      top: '10px',
-      margin: '0 0 3px 0',
-      left: centerItems ? '50%' : 25,
-      transform: 'translateX(-50%)',
-    };
-
-    const downStyle = {
-      opacity: this.props.visible ? 1 : 0,
-      bottom: '10px',
-      right: centerItems ? '50%' : 25,
-      transform: 'translateX(50%)',
-    };
-
-    const minecraftButton = {
-      width: 40,
-    };
-
-    const containerStyle = {
-      height: this.props.height,
-    };
-
     // for most tutorials, we have minimalist arrow elements. For
     // minecraft, we use a special button element to stylistically align
     // with the other buttons on the screen.
 
     const upButton = this.props.isMinecraft ? (
       <button
```

Comment:
Just confirming my understanding on why minecraft levels on production the up/down buttons are currently not appearing to be tab-navigable even though they use `<button>`. It is because they use the `outline-style: none;` - is that correct?

Reply:
Correct!